monitoring-as-code/src/metric-types.libsonnet

```diff
@@ -269,7 +269,7 @@
       },
       customSelectorLabels: {
         deadletterQueueName: 'dimension_QueueName',
-        deadletterQueueType: 'queue_type',
+        deadletterQueueType: 'dimension_queue_type',
```

Comment:
@michaelpearsonHO keen to get your thoughts on this change. To distinguish between standard and dead-letter queue types, MaC seems to check for QueueName matching or not matching `/.+dlq.+/` and a `queue_type` label either matching or not matching "deadletter".

When YACE exports metrics from AWS (at least from what we've seen from testing against LocalStack) it seems to prefix labels with `tag_` or `dimension_` depending on the Tags assigned to the resource or the dimensions of the metrics themselves. I don't see how YACE would have exported a label called "queue_type" unless it was some kind of custom metric. I don't want to change this if it's been proven to work in a real AWS environment though.

Reply:
Good spot @samiwelthomasHO. This is a hangover from from when we attempted to add an additional label to the EBSA config but they didn't want to adopt it. Looks like this is still being used in the detail dashboards so we need to refactor this in next Sprint.

Reply:
Ah okay, for now is it okay to change this to `dimension_queue_type`? In the next sprint we can possibly remove altogether if we're only going to be going on QueueName moving forwards.

Reply:
Perfect.